### PR TITLE
Added a couple simple example for using tmux or byobu to launch cmd2 applications

### PR DESCRIPTION
**Pull request by tleonhardt**
Added a couple simple example shell scripts demonstrating how to use [tmux](https://github.com/tmux/tmux/wiki) or [byobu])(https://www.byobu.org/) to launch a `cmd2` application in a terminal multiplexer along with another application such as a shell.

One example uses windows/tabs and the other uses a split screen mode.

Using a terminal multiplexer to have a `cmd2` app running at the same time as a shell within a single physical terminal is a common use case I have seen multiple `cmd2` customer use. This intent is to make this use case more discoverable for those not very familiar with `tmux` or `byobu`.

Also added a README.md with brief documentation about each example and added that to the built documentation as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,30 +39,30 @@ The price we pay for beautifully colored displays is complexity required to aggr
 The `cmd2` framework provides a great mixture of both worlds. Application designers can easily create complex applications and rely on the cmd2 library to offer effortless user facing help and extensive tab completion.
 When users become comfortable with functionality, cmd2 turns into a feature rich library enabling a smooth transition to full automation. If designed with enough forethought, a well implemented cmd2 application can serve as a boutique workflow tool. `cmd2` pulls off this flexibility based on two pillars of philosophy:
 
-- Tab Completion
-- Automation Transition
+-   Tab Completion
```
Comment (tleonhardt) [inline README.md:42]: All whitespaces like this are due to [prettier](https://prettier.io/) auto-formatting of Markdown and YAML files accoridng to the configuration in the `.prettierrc` configuration file in the `cmd2` project root directory.

There is a `Prettier` extension for VSCode and a `Prettier` plugin for PyCharm - both make it easy to auto-format on save.

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+## List of cmd2 examples
+
+Here is the list of examples in alphabetical order by filename along with a brief description of each:
+
+-   [alias_startup.py](https://github.com/python-cmd2/cmd2/blob/master/examples/alias_startup.py)
+    -   Demonstrates how to add custom command aliases and how to run an initialization script at startup
+-   [arg_decorators.py](https://github.com/python-cmd2/cmd2/blob/master/examples/arg_decorators.py)
+    -   Demonstrates how to use the `cmd2.with_argparser` decorator to specify command arguments using [argparse](https://docs.python.org/3/library/argparse.html)
+-   [arg_print.py](https://github.com/python-cmd2/cmd2/blob/master/examples/arg_print.py)
+    -   Demonstrates how arguments and options get parsed and passed to commands and shows how shortcust work
```
Comment (kmvanbrunt) [inline examples/README.md:14]: Fix "shortcuts" spelling.

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+-   [basic_completion.py](https://github.com/python-cmd2/cmd2/blob/master/examples/basic_completion.py)
+    -   Show how to enable custom tab completion by assigning a completer function to `do_*` commands
+-   [cmd2_as_argument.py](https://github.com/python-cmd2/cmd2/blob/master/examples/cmd_as_argument.py)
+    -   Demonstrates how to accept and parse command-line arguments when invoking a cmd2 application
+-   [colors.py](https://github.com/python-cmd2/cmd2/blob/master/examples/colors.py)
+    -   Show various ways of using colorized output within a cmd2 application
+-   [custom_parser.py](https://github.com/python-cmd2/cmd2/blob/master/examples/custom_parser.py)
+    -   Demonstrates how to create your own customer `Cmd2ArgumentParser`; used by the `override_parser.py` example
+-   [decorator_example.py](https://github.com/python-cmd2/cmd2/blob/master/examples/decorator_example.py)
+    -   Shows how to use cmd2's various argparse decotrators to processes command-line arguments
```
Comment (kmvanbrunt) [inline examples/README.md:30]: Fix "decorators" spelling.

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+-   [migrating.py](https://github.com/python-cmd2/cmd2/blob/master/examples/migrating.py)
+    -   A simple `cmd` application that you can migrate to `cmd2` by changing one line
+-   [modular_commands_basic.py](https://github.com/python-cmd2/cmd2/blob/master/examples/modular_commands_basic.py)
+    -   Demonstrates based `CommandSet` usage
+-   [modular_commands_dynamic.py](https://github.com/python-cmd2/cmd2/blob/master/examples/modular_commands_dynamic.py)
+    -   Demonstrates dynamic `CommandSet` loading and unloading
+-   [modular_commands_main.py](https://github.com/python-cmd2/cmd2/blob/master/examples/modular_commands_main.py)
+    -   Complex example demonstrating a variety of methods to load `CommandSets` using a mix of command decorators
+-   [modular_subcommands.py](https://github.com/python-cmd2/cmd2/blob/master/examples/modular_subcommands.py)
+    -   Shows how to use dynamically add and remove subcommands at runtime using `CommandSets`
```
Comment (kmvanbrunt) [inline examples/README.md:62]: Remove "use".

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+-   [override-parser.py](https://github.com/python-cmd2/cmd2/blob/master/examples/override_parser.py)
+    -   Shows how to override cmd2's default `Cmd2ArgumentParser` with your own customer parser class
+-   [paged_output.py](https://github.com/python-cmd2/cmd2/blob/master/examples/paged_output.py)
+    -   Shows how to use output pagination within `cmd2` apps via the `ppaged` method
+-   [persistent_history.py](https://github.com/python-cmd2/cmd2/blob/master/examples/persistent_history.py)
+    -   Shows how to enable persistent history in your `cmd2` application
+-   [pirate.py](https://github.com/python-cmd2/cmd2/blob/master/examples/pirate.py)
+    -   Demonstrates many features including colorized output, multiline commands, shorcuts, defaulting to shell, etc.
+-   [python_scripting.py](https://github.com/python-cmd2/cmd2/blob/master/examples/python_scripting.py)
+    -   Shows how cmd2's built-in `run_pyscript` command and provide advanced Python scripting of cmd2 applications
```
Comment (kmvanbrunt) [inline examples/README.md:72]: Fix wording of this sentence.

File: examples/tmux_split.sh [hunk 6]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env zsh
+
+# This script launches two applications using byobu in different tabs.
+# The user is required to enter the name of at least the first application.
+# If the second isn't provided, then the user's default shell is launched for this.
+#
+# byobu must be installed for this script to work and you can install it using your
+# operating system package manager. or info on how to use Byobu, see: https://www.byobu.org/
```
Comment (kmvanbrunt) [inline examples/tmux_split.sh:8]: Change "or" to "For".

File: examples/tmux_launch.sh [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+# This script launches two applications using tmux in different windows/tabs.
+# The user is required to enter the name of at least the first application.
+# If the second isn't provided, then the user's default shell is launched for this.
+# You must have tmux installed and that can be done using your operating system's package manager.
+#
+# See the tmux Wiki for info on how to use it: https://github.com/tmux/tmux/wiki.
+# To shift focus between different windows in tmux use Ctrl-b followed by l (lowercase "L").
+#
+# NOTE: IF you have byobu installed, it is a wrapper around tmux and will likely run instead of tmux.
```
Comment (kmvanbrunt) [inline examples/tmux_launch.sh:11]: Change "IF" to "If".

**Review by kmvanbrunt — CHANGES_REQUESTED** [review]
A few spelling and grammar errors.

**Pull request merged**